comp/forwarder/defaultforwarder/worker.go

```diff
@@ -84,7 +90,22 @@ func NewWorker(
 
 // NewHTTPClient creates a new http.Client
 func NewHTTPClient(config config.Component) *http.Client {
-	transport := httputils.CreateHTTPTransport(config)
+	var transport *http.Transport
+
+	transportConfig := config.Get("forwarder_http_protocol")
```

Comment:
New config options need to be declared in the [config package](https://github.com/datadog/datadog-agent/blob/b9382031f377c0710816729ee4734e085eccde7c/pkg/config/setup/config.go).

If an option is expected to be set by the user, it probably needs to be added to the [config template](https://github.com/datadog/datadog-agent/blob/b9382031f377c0710816729ee4734e085eccde7c/pkg/config/config_template.yaml).